copyright

```diff
@@ -499,21 +499,20 @@ Comment: Derived from works by Michael Zahniser (under the same license).
 
 Files:
  sounds/pincer*
-Copyright: Nomadic Volcano (nomadicvolcano@gmail.com)
+Copyright: David White <dave@whitevine.net>
 License: GPL-2
-Comment: Nomadic Volcano modified these from Battle for Wesnoth (https://www.wesnoth.org) sounds, which are copyright David White <dave@whitevine.net> under GPL 2 or later.
+Comment: Modified from Battle for Wesnoth (https://www.wesnoth.org) sounds, which are copyright David White <dave@whitevine.net> under GPL 2 or later.
```

Comment:
Someone should check the suitability of assigning someone random copyright for derived works created from their GPL2+ works.

Reply:
*I am not a lawyer, and this is legal advice. For legal advice, contact a lawyer.*

You have it backwards. A derived work remains the property of the work from which it was derived, unless there is a license stating otherwise. In other words, if David White used "all rights reserved," I would be violating his copyright by making a derived work and giving it to Endless Sky. David used GPL2 though, so anyone can take over the copyright.

We can change it to MZ if that would make you happier.